src/Sulu/Bundle/MarkupBundle/phpunit.xml.dist

```diff
@@ -20,4 +20,8 @@
     <php>
         <env name="APP_ENV" value="test"/>
```

Comment:
missing the `SYMFONY_DEPRECATIONS_HELPER` definition here

Reply:
Good catch! Added it :slightly_smiling_face: